libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewContext.kt

```diff
@@ -18,19 +17,21 @@ import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.map
 
 /**
- * This context is a top level data object for [DropInNavigationView].
+ * This context is a top level data object for [NavigationView] which gives access to [context]
+ * and [NavigationViewModel]. The lifecycle of this object is attached to the lifecycle of
+ * [NavigationView] and will destroy and recreate upon screen orientation or app theme changes.
```

Comment:
not sure if orientation and theme are the only way to destroy and recreate

```suggestion
 * [NavigationView] and will destroy and recreate upon configuration changes.
```